mongo/client.go

```diff
@@ -64,7 +68,16 @@ type Client struct {
 	crypt          *driver.Crypt
 }
 
-// Connect creates a new Client and then initializes it using the Connect method.
+// Connect creates a new Client and then initializes it using the Connect method. This is equivalent to calling
+// NewClient followed by Client.Connect.
+//
+// The NewClient function does not do any I/O and returns an error if the given options are invalid.
+// The Client.Connect method starts background goroutines to monitor the state of the deployment and does not do
+// any I/O in the main goroutine to prevent the main goroutine from blocking. Therefore, it will not error if the
+// deployment is down.
+//
+// The Client.Ping method can be used to verify that the deployment is successfully connected and the
```

Comment:
Have you considered whether the documentation from NewClient should be copied here?  It seems as relevant here as it is there, and iirc this is the more commonly used method of the two.

Reply:
copied it into the `Connect` docs right above the part about `Connect` not erroring if the deployment is down.